docs/operate/reference/advanced-modules/create-subtype.md

```diff
@@ -106,3 +106,15 @@ Now that your resource API is defined, create a new model that implements your n
 
 {{% /manualcard %}}
 {{< /cards >}}
+
+{{% alert title="Note" color="note" %}}
+
+After you define a new API and create a model that implements it, keep the following in mind when writing code against your new API:
+
+- You can't use [SDKs](/dev/reference/sdks/) to call your new API unless you build out the SDKs to support it.
+  It is easiest to write code against your new API in the language you used to define it.
+- Since your API doesn't have built-in SDK support, you'll need a local copy of the module code on whatever machine is running client code against it.
+- Be sure to import the API definition from the module directory, for example `from /path/to/module/src.gizmo import Gizmo`.
```

Comment:
the import reads a bit like pseudo-code. Should be more like `from path.to.module import Gizmo` if we're doing python

Reply:
```suggestion
- Be sure to import the API definition from the module directory, for example in Python it would be of the form `from path.to.module.src.gizmo import Gizmo`.
```